backend/validators/llm.py

```
@@ -0,0 +1,141 @@
+            "--config",
+            self._config.new_path,
+            "--allow-empty-backends",
+            "--die-with-parent",
+            stdin=None,
+            stdout=sys.stdout,
+            stderr=sys.stderr,
+        )
+
+    async def verify_for_read(self):
+        if self._process is None:
+            raise Exception("process is not started")
```

Pylint (3.3.7)

[warning] Raising too general exception: Exception

(W0719)